synadm/api.py

```diff
@@ -55,6 +56,8 @@ def __init__(self, log, user, token, base_url, path, timeout, debug):
                 base_url to form the basis for all API endpoint paths
             timeout (int): requests module timeout used in query method
             debug (bool): enable/disable debugging in requests module
+            verify(bool): SSL verification is turned on by default
+                and can be turned off using this method.
```

Comment:
I'd prefer the wording: ....using this argument.

Also, is the type now still bool?

Reply:
Good question! I'll check

Reply:
We can keep bool type